Frame.cc

```diff
@@ -730,7 +744,7 @@ void Frame::SetImageCache() {
     casacore::Slicer section = GetChannelMatrixSlicer(_channel_index, _stokes_index);
     casacore::Array<float> tmp(section.length(), _image_cache.data(), casacore::StorageInitPolicy::SHARE);
     std::lock_guard<std::mutex> guard(_image_mutex);
-    _loader->LoadData(FileInfo::Data::Image)->getSlice(tmp, section, true);
+    _loader->GetSlice(tmp, section, false);
```

Comment:
Has the flag value been changed here deliberately, or is this a C&P error?

Reply:
Changed deliberately. I was getting a casacore Array conformance error (shapes did not match) when degenerate axes were removed.